commit b609280fc0d0f0f5a728b08e874977cb16f921b8
Author: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>
Date:   Thu May 2 08:17:42 2019 +0200

    0001 Drop GCC check
    
    Skip GCC badly written GCC check intended for older released that may cause
    bugs during future GCC updates
    
    Signed-off-by: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>

diff --git autogen.sh autogen.sh
index a0b3d897d15b..f13f3ad543f0 100755
--- autogen.sh
+++ autogen.sh
@@ -516,7 +516,7 @@ fi
 
 echo_n "Checking for automake version... "
 recreate_tmp
-ver=1.15
+ver=1.13
 cat > .tmp/configure.ac<<EOF
 AC_INIT(testver,1.0)
 AC_CONFIG_AUX_DIR([m4])
@@ -554,7 +554,7 @@ fi
 
 echo_n "Checking for libtool version... "
 recreate_tmp
-ver=2.4.4
+ver=2.4.2
 cat <<EOF >.tmp/configure.ac
 AC_INIT(testver,1.0)
 AC_CONFIG_AUX_DIR([m4])
diff --git src/binding/cxx/buildiface src/binding/cxx/buildiface
index 3afd823c8bd0..9f73c84d7eb8 100755
--- src/binding/cxx/buildiface
+++ src/binding/cxx/buildiface
@@ -1454,21 +1454,6 @@ sub printDefineChecks {
 #endif\n\n";
     }
 
-    # GCC changed the calling convention between 3.2.3 and 3.4.3 (!!!)
-    # check for that
-    print $OUTFD "
-// Check for incompatible GCC versions
-// GCC (specifically) g++ changed the calling convention
-// between 3.2.3 and 3.4.3 (!!)  Normally such changes
-// should only occur at major releases (e.g., version 3 to 4)
-#ifdef __GNUC__ 
-# if __GNUC__ >= \@GNUCXX_VERSION\@ 
-#  if __GNUC_MINOR__ > 2 && \@GNUCXX_MINORVERSION\@ == 2 
-#  error 'Please use the same version of GCC and g++ for compiling MPICH and user MPI programs'
-#  endif
-# endif     
-#endif\n";
-
     print $OUTFD "
 /* 
  * Because the API is defined, some methods have parameters that are 
